src/handlers.ts

```diff
@@ -0,0 +1,52 @@
+import 'source-map-support/register';
+import path from 'path';
+import fs from 'fs';
+import { readOperationsRepository } from '@api3/operations/dist/utils/read-operations';
+import { logging, opsGenie, promises } from '@api3/operations-utilities/dist/index';
+import { runWalletTasks } from './wallet-metrics';
+import { WalletConfig } from './types';
+
+export const getWalletConfig = (): WalletConfig => {
+  const configPath = path.join(__dirname, '../config/walletConfig.json');
+  logging.debugLog('Config Path:', configPath, fs.readdirSync(path.join(__dirname, '..')));
+
+  return JSON.parse(fs.readFileSync(configPath).toString('utf-8'));
+};
+
+/**
+ * Tops up wallets
+ *
+ * @param _event
+ */
+export const walletTasksHandler = async (_event: any = {}): Promise<any> => {
+  console.log('Starting...');
+  const startedAt = new Date();
+  const walletConfig = getWalletConfig();
+  await opsGenie.cacheOpenAlerts(walletConfig.opsGenieConfig);
+
+  const [err] = await promises.go(
+    async () => {
+      const opsConfig = readOperationsRepository();
+
+      await runWalletTasks(walletConfig, opsConfig);
+    },
```

Comment:
```suggestion
   () => {
      const opsConfig = readOperationsRepository();

      return runWalletTasks(walletConfig, opsConfig);
    },
```
I'm curious, would this work?